TestTransactionTimeoutSecondStatement is flaky

```
--- FAIL: TestTransactionTimeoutSecondStatement (0.10s)
    transaction_test.go:336: error code mismatch
         Got: OK
        Want: DeadlineExceeded
```

--- a/transaction_test.go
+++ b/transaction_test.go
@@ -327,7 +327,7 @@ func TestTransactionTimeoutSecondStatement(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	server.TestSpanner.PutExecutionTime(testutil.MethodExecuteStreamingSql, testutil.SimulatedExecutionTime{MinimumExecutionTime: 30 * time.Millisecond})
+	server.TestSpanner.PutExecutionTime(testutil.MethodExecuteStreamingSql, testutil.SimulatedExecutionTime{MinimumExecutionTime: 50 * time.Millisecond})
 	rows, err := tx.QueryContext(ctx, testutil.SelectFooFromBar, ExecOptions{DirectExecuteQuery: true})
 	if rows != nil {
 		_ = rows.Close()

